retriever/lib/engine.py

```diff
@@ -488,11 +490,21 @@ def download_file(self, url, filename):
                     hooks={'response': reporthook(progbar, path)},
                 )
 
+                if response.status_code == 404:
+                    print ("404 error: The data source not found, or server not found ")
```

Comment:
Remove the space after `print`.
This message seems more appropriate.
```
Error 404: The data source or server not found
```

Reply:
Removed! I will notice this small mistake next time.
Thx!

Reply:
Travis courrpted, but the code should be correct.

Reply:
@coolalexzb Can you also update the print message to `Error 404: The data source or server not found`

Reply:
Megered the master branch and updated the msg. Thx! @apoorvaeternity